slack_approval/slack_provision.py

```diff
@@ -19,7 +19,6 @@ def __init__(self, request, requesters_channel=None):
         self.inputs = json.loads(action["value"])
         self.ts = self.inputs.pop("ts")
         self.requesters_channel = self.inputs.pop("requesters_channel")
-        self.approvers_channel = self.inputs.pop("approvers_channel")
```

Comment:
hmm i think we can still keep this. just need to check if it exists or not.
see programiz.com/python-programming/methods/dictionary/pop